diff --git a/src/gallium/frontends/rusticl/meson.build b/src/gallium/frontends/rusticl/meson.build
index eb87dad2659..f2452501aae 100644
--- a/src/gallium/frontends/rusticl/meson.build
+++ b/src/gallium/frontends/rusticl/meson.build
@@ -218,6 +218,7 @@ rusticl_llvm_bindings_rs = rust.bindgen(
   c_args : [
     rusticl_bindgen_c_args,
     pre_args,
+    '-std=c++17',
   ],
   dependencies : [
     dep_clang,
